### ISSUE TITLE
NTR: MHPG low in CSF (Lab: cerebrospinal fluid tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8447]
New term request MHPG low in CSF (segal_080211201631): 3-methoxy-4-hydroxy-phenylethyleneglycol; cerebrospinal fluid  UMLS: 2676448

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000398
- label: Decreased CSF 3-methoxy-4-hydroxyphenylglycolaldehyde concentration
- parents: [Abnormal CSF metabolite concentration (HP:0025454)](https://hpo.jax.org/app/browse/term/HP:0025454)
- pmid: PMID:6539879
- def: The concentration of 3-methoxy-4-hydroxyphenylglycolaldehyde in the cerebrospinal fluid (CSF) is below the lower limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8447
- synonyms:
